internal/cli/termui/ui.go

```diff
@@ -108,10 +108,11 @@ func (u *UI) DisableJSON() {
 	u.jsonEnabled = false
 }
 
-func (u *UI) JSON(value any) {
+func (u *UI) JSON(value any) bool {
 	if u.jsonEnabled {
 		_ = json.NewEncoder(u.output).Encode(value)
 	}
+	return u.jsonEnabled
 }
```

Comment:
This makes sense, but maybe it's nicer to separate the two. Since the encoder is also returning an error that we are ignoring it's probably better to do something like this:

```go
func (u *UI) JSON(value any) error {
	if u.jsonEnabled {
		return json.NewEncoder(u.output).Encode(value)
	}
	return nil
}

func (u *UI) JSONEnabled() bool {
	return u.jsonEnabled
}
```

and then

```go
if c.ui.JSONEnabled() {
	// Avoid doing all the formatting for human consumption which will not be shown due to json mode
	return c.ui.JSON(configurations)
}
```

Reply:
done with commit 89e4d35b1c0c7a5b15aef82a515c9d95091694c2